packages/cli/src/modules/data-table/data-table.controller.ts

```diff
@@ -261,6 +261,36 @@ export class DataTableController {
 		}
 	}
 
+	@Get('/:dataTableId/download-csv')
+	@ProjectScope('dataTable:read')
+	async downloadDataTableCsv(
```

Comment:
Can we add a integration test for this endpoint? that will automatically add coverage to all the backend code (or most of it)